notebooks/Viewing 3D volumetric data with matplotlib.ipynb

```diff
@@ -54,7 +61,10 @@
    },
    "outputs": [],
    "source": [
+    "# import data module \n",
     "from skimage import data\n",
+    "\n",
+    "# Gather some data \n",
```

Comment:
This one is better, but I would rather place it in the markdown cell above it, as follows:

"""
Now we can import matplotlib and display some data. We'll load some example data that is included in the `data` module of the *scikit-image* library:
"""